src/components/dashboard/container/OwnerDictionary.jsx

```diff
@@ -58,14 +62,17 @@ export class OwnerDictionary extends Component {
     const { searchInput } = this.state;
     return (
       <div className="dashboard-wrapper">
-        <UserDashboard
-          dictionaries={dictionaries}
-          organizations={organizations}
-        />
+        <Title title="Login" />
+        <UserDashboard dictionaries={dictionaries} organizations={organizations} />
         <p> Your dictionaries </p>
         <div className="dashboard-head">
-          <div className="btn add-dictionaries" onClick={this.handleShow}>
-              Add Dictionary
+          <div
+            className="btn add-dictionaries"
```

Comment:
Do all these changes have anything to do with the ticket?

Reply:
Same here. Let me revert these unrelated changes.